windows/start.sh

```diff
@@ -68,4 +68,10 @@ docker () {
 }
 export -f docker
 
-exec "${BASH}" --login -i
+if [ $# -eq 0 ]; then
+  echo "Start interactive shell"
+  exec "$BASH" --login -i
+else
+  echo "Start shell with command"
+  exec "$BASH" -c "$1"
```

Comment:
Hmm, `$*` might actually be more appropriate than either `$1` or `$@` here, so that we're explicit about our space expansion.